app/src/main/java/com/cornellappdev/android/eatery/MapsActivity.java

```diff
@@ -33,12 +33,16 @@
 import java.util.ArrayList;
 
 public class MapsActivity extends AppCompatActivity implements OnMapReadyCallback {
-  private GoogleMap mMap;
-  ArrayList<EateryBaseModel> cafeData;
   public Repository repositoryInstance = Repository.getInstance();
-  private static final int LOCATION_PERMISSION_REQUEST_CODE = 1;
   public BottomNavigationView bnv;
 
+  private static final int LOCATION_PERMISSION_REQUEST_CODE = 1;
```

Comment:
Why do you do this in the Map activity and Main activity? The fragments should be created once and reused everywhere. It will increase performance

Reply:
I'm not sure if sharing fragments across two activities is architecturally(?) correct--or at least I've never seen any examples that do so. The map activity really should be a fragment, especially since the map and main activity share the same bottom nav bar.